api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -83,4 +91,7 @@ message DnsCacheConfig {
   // this is used as the cache's DNS refresh rate when DNS requests are failing. If this setting is
   // not specified, the failure refresh rate defaults to the dns_refresh_rate.
   config.cluster.v3.Cluster.RefreshRate dns_failure_refresh_rate = 6;
+
+  // The config of circuit breakers for resolver. It provides a configurable threshold.
+  DnsCacheCircuitBreakers dns_cache_circuit_breaker = 7;
```

Comment:
comment about what happen this is not set. (it is different from the `max_pending_requests` is not set)